Makefile

```diff
@@ -244,7 +244,9 @@ endef
 define CI_ENV_VARS
 export OP_TESTLOG_FILE_LOGGER_OUTDIR=$$(realpath ./tmp/testlogs) && \
 export SEPOLIA_RPC_URL="https://ci-sepolia-l1-archive.optimism.io" && \
-export MAINNET_RPC_URL="https://ci-mainnet-l1-archive.optimism.io"
+export MAINNET_RPC_URL="https://ci-mainnet-l1-archive.optimism.io" && \
+export NAT_INTEROP_LOADTEST_TARGET=10 && \
+export NAT_INTEROP_LOADTEST_TIMEOUT=30s
```

Comment:
Is this expected to be in here? Seems sensible, just checking.

Reply:
Josh reminded me that we may in fact run load tests in CI (via `go test` or `gotestsum`), so we should probably have these in CI.

FYI: I'm working on isolating all acceptance tests under the one workflow/job (and so that they all run with `op-acceptor`). I'll move/remove these as part of that work if needed. 
https://github.com/ethereum-optimism/optimism/pull/16755

Reply:
From a slack discussion: it appears we are running the load tests in CI. The small parameters look good. We should aim to ensure the load tests function properly without actually doing full-blown load tests.

The `NAT_INTEROP_LOADTEST_TIMEOUT=30s` is a tad suspect, since the test may not be set up in that amount of time, especially if the CI box is slow. But I would vote for not optimizing this here.

Reply:
Ah, github didn't reload so I didn't see Stefano's message. All well.